src/main/java/fi/pohina/vinkkilista/domain/Blog.java

```diff
@@ -0,0 +1,75 @@
+package fi.pohina.vinkkilista.domain;
+
+import java.util.*;
+
+public class Blog implements Bookmark {
+
+    private String title;
+    private String author;
+    private String url;
+    private BookmarkType type;
+    private ArrayList<Course> relatedCourses;
+
+    /**
+     * Constructor
+     * @param title
+     * @param url
+     * @param type 
+     */
+    public Blog(String title, String url, BookmarkType type) {
```

Comment:
Probably don't need to pass the `BookmarkType` in the ctor, since a `Blog` is always a `Bookmark` with the type `BookmarkType.Blog`